devel/ci/bodhi-ci

```diff
@@ -886,6 +977,10 @@ def _run_jobs(jobs):
     Args:
         jobs (list): A list of Jobs to run.
     """
+    if not jobs:
+        click.echo("No jobs!")
+        sys.exit(3)
```

Comment:
Heh, is there a way to get this to happen?

Reply:
Yeah, if you ask for integration testing only and for an unsupported fedora release version.